ci-operator/step-registry/openshift-extended/web-tests/openshift-extended-web-tests-ref.yaml

```diff
@@ -6,7 +6,7 @@ ref:
   timeout: 8h0m0s
   resources:
     requests:
-      cpu: "2"
+      cpu: "500m"
```

Comment:
I saw most of vsphere step are limited to `100m`, can we also set it to `100m` here?

Reply:
I'm fine with it. Done